modules/core/nuxeo-core-bulk/src/main/java/org/nuxeo/ecm/core/bulk/message/BulkStatus.java

```diff
@@ -82,6 +84,9 @@
     @Nullable
     protected String errorMessage;
 
+    /** @since 11.5 **/
+    protected int errorCode;
```

Comment:
For old BulkStatus presents in streams, this field should be an Integer and have a `@Nullable` annotation.